### ISSUE TITLE
Bug: guild compare crashes when sorting by column for which some runs don't have values

### ISSUE THREAD

**Issue by FelixAbrahamsson** [guildai/guildai#81]
Getting the following error when I do `guild compare -c best_loss -min best_loss` when a value for best_loss doesn't exist for some of the runs
```
TypeError: '<' not supported between instances of 'NoneType' and 'NoneType'
```
Interestingly, I do not get the error when I run  `guild compare -min best_loss`, it only happens if I specify columns with the -c flag (which I'm combining with the -n flag in my use case).
I submitted a pull request #80 with a proposed fix.

**Comment by gar1t** [comment]
Thank you - my apologies for the late reply here. I did merge #80. I'll make sure this is addressed in 0.7.

**Comment by gar1t** [comment]
This is fixed in master and will be available in 0.7.0.rc10. Thanks again for the help!

**Comment by gar1t** [comment]
Fix is now available in 0.7.0.
